sdk/nodejs/automation/localWorkspace.ts

```diff
@@ -721,6 +721,14 @@ export class LocalWorkspace implements Workspace {
         const result = await this.runPulumiCmd(["stack", "ls", "--json"]);
         return JSON.parse(result.stdout);
     }
+    /**
+     * Returns all Stacks the current logged-in Pulumi identity has access to.
+     * This queries underlying backend and may return stacks not present in the Workspace (as Pulumi.<stack>.yaml files).
+     */
+    async listAllStacks(): Promise<StackSummary[]> {
+        const result = await this.runPulumiCmd(["stack", "ls", "--json", "--all"]);
+        return JSON.parse(result.stdout);
+    }
```

Comment:
This is small enough where no shared helper is necessary, IMO. So looks good as-is!